docs/admin/backup.rst

```diff
@@ -173,37 +172,36 @@ Database
 
 The actual storage location depends on your database setup.
 
-The database is the most important storage. Set up regular
-backups of your database, without it all your translation setup will be gone.
+The database is the most important thing to have regular backups of.
+Without it all the work going into translation is gone.
 
 Native database backup
 ++++++++++++++++++++++
 
-The recommended approach is to do dump of the database using database native
+The recommended way is to do save a dump of the database using database native
 tools such as :program:`pg_dump` or :program:`mysqldump`. It usually performs
-better than Django backup and restores complete tables with all data.
+better than Django backup, and it restores complete tables with all their data.
 
-You can restore this backup in newer Weblate release, it will perform any
+You can restore this backup in newer Weblate releases, where it performs all
```

Comment:
Since version?

Reply:
The database migrations are there since ever.